### DISCUSSION TITLE
Running carla with gpu-operator on kubernetes

### DISCUSSION THREAD

**Discussion by DirkTheDaring** [carla-simulator/carla#5437] [General]
Is there any helm chart to deploy carla?

Has anybody success in running carla on a kubernetes cluster together with the nvidia [gpu-operator]?(https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/overview.html)?

**Comment by JasonBourne1998** [comment]
Have you tried this idea @DirkTheDaring ?

**Reply by DirkTheDaring** [reply]
yes. works

**Comment by WaelKarkoub** [comment]
Hi @DirkTheDaring, will you be able to share how you were able to run Carla on Kubernetes?

**Reply by mickume** [reply]
@DirkTheDaring  I would also be interested to know more, we're currently looking into running Carla on Red Hat OpenShift ...

**Reply by OA-1234** [reply]
@DirkTheDaring I am also interested in knowing more

**Reply by anilcanbulut** [reply]
Any updates? I am also interested in running CARLA server inside K8s cluster

**Comment by kallegrens** [comment]
We are looking to do this too! Any helm chart available?
